### PR TITLE
After demo optimizations

### PR DESCRIPTION
**Pull request by EricMason24**
Fixed some tech debt from demo + added in a feature request from tara to see the available spaces in a breakout session/ sign up for a waitlist if a breakout session is at max capacity.

### CODE REVIEW

File: src/components/SelectBreakout.vue [hunk 1]
```diff
@@ -50,6 +50,13 @@ export default {
     timeSlot: String
   },
   computed:{
+    computedDataRanking() {
```
Comment (dborstelmann) [inline src/components/SelectBreakout.vue]: NICE

Comment (dborstelmann) [reply src/components/SelectBreakout.vue]: Jedi apprentice is learning

File: src/components/Session.vue [hunk 2]
```diff
@@ -8,6 +8,8 @@
         <p></p>
       </div>
       <h2 class="description">{{description}}</h2>
+      <h3 class="description" v-if="computedSpace > 0">Spaces Available: {{computedSpace}}</h3>
```
Comment (dborstelmann) [inline src/components/Session.vue]: What about 1 “space” available?

File: src/views/SponsorPage.vue [hunk 3]
```diff
@@ -3,8 +3,8 @@
   <NavBar></NavBar>
     <h1>Sponsors</h1>
     <h3>Description of Our Sponsors</h3>
-    <div class>
-      <el-button class="btn">Email Us About Becoming A Sponsor</el-button>
+    <div class="emailLink">
+      <a href="mailto:eric.mason@slalom.com">Email Us About Becoming A Sponsor</a>
```
Comment (dborstelmann) [inline src/views/SponsorPage.vue]: You gonna change that email?

Comment (EricMason24) [reply src/views/SponsorPage.vue]: Yeah, just put it in because we don't have what email we want

File: src/components/Session.vue (hunk 2 above)
Comment (EricMason24) [reply src/components/Session.vue]: fixed 👍

**Pull request merged**